src/olympia/api/exceptions.py

```diff
@@ -4,13 +4,20 @@
 from django.core.exceptions import PermissionDenied
 from django.core.signals import got_request_exception
 from django.http import Http404
+from django.utils.translation import ugettext_lazy as _
 
 from rest_framework import exceptions
 from rest_framework import status
 from rest_framework.response import Response
 from rest_framework.views import set_rollback
 
 
+class UnavailableForLegalReasons(exceptions.APIException):
+    status_code = status.HTTP_451_UNAVAILABLE_FOR_LEGAL_REASONS
+    default_detail = _('Unavailable for legal reasons.')
+    default_code = 'unavailable_for_legal_reasons'
```

Comment:
add the Link header as suggested?  https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/451  (it's a "SHOULD" in https://tools.ietf.org/html/rfc7725#section-4)

Reply:
+1

Reply:
I can look into that, but I'm not sure where to link to. https://www.mozilla.org/en-US/about/policy/transparency/ ?

Reply:
Sounds good.

Reply:
ffb10974610e186d7f8cfeff99de2f88d4bfd01b